cmd/main.go

```diff
@@ -162,7 +158,17 @@ func main() {
 			log.Fatal("failed to create client:", err)
 		}
 		generator = job.New(log.WithField("pkg", "k8s-job-wrapper"), kclient, Options.JobConfig)
+
+		cfg, cerr := clientcmd.BuildConfigFromFlags("", "")
+		if cerr != nil {
+			log.Fatalf("Failed to create kubernetes cluster config")
```

Comment:
please add the error here as well.

Reply:
if u mean "var kclient client.Client" it is not, it is runtime client, it is different

Reply:
and then what? currenly it's gonna log and continue?

Reply:
nope
it will exit

Reply:
u can't run without leader if u  want 3 instances. Better to know it immediately

Reply:
i was talking about `log.WithError(cerr).Fatal("Failed to create kubernetes cluster config")`